dvol_python/dvol.py

```diff
@@ -166,9 +182,13 @@ def createBranch(self, volume, branch):
         self.output("Created branch %s/%s" % (volume, branch))
 
     def createVolume(self, name):
-        if self._directory.child(name).exists():
-            self.output("Error: volume %s already exists" % (name,))
-            raise VolumeAlreadyExists(name)
+        try:
+            if self._directory.child(name).exists():
+                self.output("Error: volume %s already exists" % (name,))
+                return
+        except InsecurePath:
```

Comment:
:question: Ideally, the `except` would only wrap the `self._directory.child` call and nothing else, in case other things raise this error for different reasons. Not really essential here, because the code is so simple.